.github/versions.json

```diff
@@ -71,6 +71,9 @@
             "6.2",
             "6.3"
         ],
+        "symfony/bandwidth-notifier": [
+            "6.3"
```

Comment:
Not sure if it is needed to update this file manually 🧐

Reply:
I copied what was done in this [PR](https://github.com/symfony/recipes/pull/1147/files).

Reply:
You were right. Other older Notifier PRs did not change that file. I removed the change from all four of my PRs.